From: Andrea Bolognani <abologna@redhat.com>
Subject: [libvirt PATCH v4 19/19] fixup! NEWS: Mention Apple Silicon support
	for HVF
Date: Fri,  7 Jan 2022 10:47:46 +0100
Content-Type: text/plain; charset="utf-8"

And simplify the entry by removing mentions of QEMU 2.12 and
macOS 10.10, which are both ancient at this point, and of
specific hardware features that are going to be present in any
recent Apple machine.

Signed-off-by: Andrea Bolognani <abologna@redhat.com>
---
 NEWS.rst | 7 ++-----
 1 file changed, 2 insertions(+), 5 deletions(-)

diff --git NEWS.rst NEWS.rst
index 56efaa9f72..85cbdd154f 100644
--- NEWS.rst
+++ NEWS.rst
@@ -55,11 +55,8 @@ v8.0.0 (unreleased)
 
   * qemu: Add hvf domain type for Hypervisor.framework
 
-    QEMU introduced experimental support of Hypervisor.framework
-    since 2.12.
-
-    It's supported on machines with Intel VT-x feature set that includes
-    Extended Page Tables (EPT) and Unrestricted Mode since macOS 10.10.
+    It works on Intel machines as well as recent machines powered by Apple
+    Silicon. QEMU 6.2.0 is needed for Apple Silicon support.
 
 
 * **Improvements**
-- 
2.31.1
